README.md

```diff
@@ -82,6 +90,25 @@ dev_dependencies:
 }
 ```
 
+### Builder Options
+
+To configure options for the builder see the `build_config` 
+[README](https://github.com/dart-lang/build/blob/master/build_config/README.md). 
+
+* `outputStyle`: Supports `expanded` or `compressed`. Defaults to `expanded`.
+
+Example:
+
+```yaml
+targets:
+  $default:
+    builders:
+      sass_builder:
+        options:
+          outputStyle: compressed
+```
+
```

Comment:
in what file should we put this options? `build.yaml` or `sass_builder.build.yaml`?